src/app/api/tasks/tasks.service.ts

```diff
@@ -309,7 +310,8 @@ export class TasksService extends BaseService {
   }
 
   async clientUpdateTask(id: string, targetWorkflowStateId?: string | null) {
-    //Apply custom authorization here. Policy service is not used because this api is for client's Mark done function only. Only clients can use this.
+    // Apply custom authorization here. Policy service is not used because this api is for client's Mark done
+    // function only. Only clients can use this.
```

Comment:
is this intentional to put in next line?

Reply:
Yes. In my text editor long lines like these mess up the editor's viewport